packages/dd-trace/src/appsec/iast/analyzers/cookie-analyzer.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const Analyzer = require('./vulnerability-analyzer')
+
+const EXCLUDED_PATHS = ['node_modules/express/lib/response.js', 'node_modules\\express\\lib\\response.js']
+
+class CookieAnalyzer extends Analyzer {
+  constructor (type, propertyToBeSafe) {
+    super(type)
+    this.propertyToBeSafe = propertyToBeSafe.toLowerCase()
+    this.addSub('datadog:iast:set-cookie', (cookieInfo) => this.analyze(cookieInfo))
+  }
+
+  _isVulnerable ({ cookieProperties, cookieValue }) {
+    return cookieValue && !(cookieProperties && cookieProperties
+      .map(x => x.toLowerCase().trim()).includes(this.propertyToBeSafe))
+  }
+
+  _getEvidence ({ cookieName }) {
+    return { value: cookieName }
+  }
+
+  _createHashSource (type, evidence, location) {
+    return `${type}:${evidence.value}`
+  }
+
+  _getExcludedPaths () {
+    return EXCLUDED_PATHS
+  }
+  _checkOCE (context, value) {
+    if (value && value.location) {
+      return true
+    }
+    return super._checkOCE(context)
```

Comment:
pass `value` to super call?

```suggestion
    return super._checkOCE(context, value)
```